spec/components/decision_overall_result_component_spec.rb

```diff
@@ -14,17 +14,17 @@
 
     context 'when funding is "granted"' do
       let(:funding_decision) { 'granted' }
-      let(:overall_result) { 'Granted - Passed Means Test' }
+      let(:overall_result) { Types::OverallResult['granted_with_contribution'] }
 
       it { is_expected.to have_text('Granted') }
```

Comment:
Shouldn't we expect the text to be "Granted - with contribution"?

Reply:
Great catch, thank you! I have fixed string and added `exact: true` option to the matcher.